### DISCUSSION TITLE
Data Upload on BREAD module

### DISCUSSION THREAD

**Discussion by otheriz** [thedevdojo/voyager#5469] [Q&A]
### Description of problem

With so much data that has to be input..it will take a long time to input all data one by one.

### Proposed solution

It's been a very great feature with BREAD...it would be more perfect to provide the upload by CSV, excel, or any possible file..

Because most of the users have their own data by excel file that they need to input into the system..it would be helpful if it provide the upload data on the BREAD module

**Comment by pro-cms** [comment]
hi, you can download laravel excel package https://github.com/Maatwebsite/Laravel-Excel, install using the procedure required there, create your bread, create a custom view and controller as explained from voyager documentation where you will add an option button/input or modal to upload CSV, Excel etc, then handle processing data in your custom controller as explained in the package.

check : https://voyager-docs.devdojo.com/customization/overriding-files and

**Comment by vanchao0519** [comment]
Hi @otheriz ! You can try this package for data upload on BREAD module.

check: https://github.com/vanchao0519/VoyagerDataTransport
